src/lib/args.ts

```diff
@@ -0,0 +1,143 @@
+// Copyright © 2022 Relay Inc.
+
+import { TaskArgs, TaskGroupMembers } from "./api"
+
+type StartArgs = {
+  namespace: string,
+  type: string,
+  major: string,
+  name: string,
+  assignTo: string,
+  args: string | TaskArgs,
+}
+
+type ScheduleArgs = StartArgs & {
+  start: string,
+  timezone: string
+}
+
+type CreateTaskGroupArgs = {
+  namespace: string,
+  name: string,
+  type: string,
+  major: string,
+  assignTo: string,
+  members: string | TaskGroupMembers
+}
+
+const createScheduleArgs = (args: string[]): ScheduleArgs => {
+  const scheduleArgs: ScheduleArgs = {
+    namespace: args[0] as string,
+    type: args[1] as string,
+    major: args[2] as string,
+    name: args[3] as string,
+    assignTo: args[4] as string,
+    args: args[5] as string,
+    start: args[6] as string,
+    timezone: args[7] as string
+  }
+  return scheduleArgs
+}
+
+const createStartArgs = (args: string[]): StartArgs => {
```

Comment:
There is a fun utility function from lodash `zipObject`...

```
const createStartArgs = (args: string[]): StartArgs => {
  return zipObject([`namespace`, `type`,`major`, `name`, `assignTo`, `args`], args)
}
```
https://lodash.com/docs/4.17.15#zipObject